### ISSUE TITLE
Run pact broker in a subfolder

### ISSUE THREAD

**Issue by wideklev** [DiUS/pact_broker-docker#60]
Running pact-broker in our kubernetes environment and works very well and have managed very nicely to deploy it using our own helm template.

The downside is that with our template all our service are deployed in a subfolder.

All links in the pact broker are referenced with the urls starting with / which makes all links broken.

It would more reliable if all links including css and js to be relative instead of coded with / to support both running in an own domain or in an subfolder.

**Comment by bethesque** [comment]
I can't quite understand your set up from your explanation above. Are you talking about the URL paths, or the file paths? Can you give me some text examples please.

**Comment by wideklev** [comment]
We are deploying the pact-broker in a subfolder:
So our url would be something like: example.com/pact-broker:
In that context the urls are still:

`<link href='/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/stylesheets/index.css' rel='stylesheet'>
  <script src='/javascripts/jquery-2.1.1.min.js' type='text/javascript'></script>
  <script src='/javascripts/jquery.tablesorter.min.js' type='text/javascript'></script>
  <script src='/js/bootstrap.min.js' type='text/javascript'></script>
  <nav class='navbase-default navbar-right' role='navigation'>
  <div class='container'>
  <ul>
  <li class='navbar-right'>
  <a href='/hal-browser/browser.html'>`

And they should be relative ./ instead of /

Maybe the issue should be in this project instead: https://github.com/pact-foundation/pact_broker

**Comment by bethesque** [comment]
Yep, it belongs in the pact_broker project. I don't have time to look at it right now, but if you know where to make the changes (and they look quite simple) I'm happy to accept a PR.

**Comment by chris-fa** [comment]
@wideklev did you figure this out?

**Comment by bethesque** [comment]
The URLs would need to be modified in more than just the place listed above. I'm not even sure of what would need to be changed for this to work.

**Comment by somnath157** [comment]
Did you managed to find solution for it? i am facing same issues
